docs/restic.md

```diff
@@ -108,21 +112,22 @@ You're now ready to use Ark with restic.
     kubectl -n foo annotate pod/sample backup.ark.heptio.com/backup-volumes=pvc-volume,emptydir-volume
     ```
 
-
-    This annotation can also be provided in the pod template spec if using a deployment, daemonset, etc.
+    This annotation can also be provided in the pod template spec if you're using a deployment or daemonset
```

Comment:
I'd update this to be "a deployment, daemonset, or replicaset"

Reply:
Also works for replicationcontrollers, jobs, statefulsets... hence the original "etc"

Reply:
Good point. Perhaps, "This annotation can also be provider in a pod template spec."

Since the list is pretty exhaustive we don't want to list them all, but I also want to make sure we're not giving the false impression that it only supports the kinds explicitly listed.

Reply:
I'd either leave as-is, i.e. "deployment, daemonset, etc.", or say something like "in the pod template spec if you're using a controller to manage your pods"